IFC4x3/Properties/s/ShieldConductorDiameter_1as$loCX107Pk6vf9GXkJu/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="ShieldConductorDiameter_1asloCX107Pk6vf9GXkJu" Name="ShieldConductorDiameter" UniqueId="64dbfbf2-3210-401d-9b86-e6925086e4f8" PrimaryDataType="IfcPositiveLengthMeasure" />
```

Comment:
See comment about generalizing diameters.

Reply:
Ok with keeping this diameter.